### DISCUSSION TITLE
The commit should not trigger a release even with feat(component) or fix(component)

### DISCUSSION THREAD

**Discussion by tommaton** [semantic-release/semantic-release#2589] [Commit Convention]
# Current behaviour
We've got our releases working if all our commits that begin with `feat:` or `fix:` etc but when there seems to be a mixture (`feat:` and `fix:`) then there isn't a release.

No releases are generated despite a commit with `feat(component): ...` and/or `fix(component): ...`  being present in the commit messages

```
[semantic-release] [@semantic-release/commit-analyzer] › ℹ  Analyzing commit: Updating components (#216)

* feat(componentName): new feature

* fix(componentName): adding a fix to the component in PR
[semantic-release] [@semantic-release/commit-analyzer] › ℹ  The commit should not trigger a release
```

# Expected behaviour
A release is generated increasing major version.

# Environment
semantic-release version:
CI environment: Github Actions
Plugins used: ["@semantic-release/commit-analyzer", "@semantic-release/release-notes-generator", "@semantic-release/changelog", "@semantic-release/npm", "@semantic-release/git", "@semantic-release/github"]

**Comment by travi** [comment]
it looks like you squashed a PR that contained multiple commits and github's default squash behavior listed the subjects of those commits in the body of the resulting squash commit. semantic-release acts on commits. it knows nothing about the PR that resulted in a commit. it does not read commits listed in the body of a commit. this is explained further in our docs: https://semantic-release.gitbook.io/semantic-release/support/troubleshooting#squashed-commits-are-ignored-by-semantic-release

> A release is generated increasing major version.

even if semantic-release did read the commits listed in the body of another commit, neither `feat` nor `fix` would result in increasing the major version. `feat` would result in increasing the minor version.
